### PR TITLE
Check for grammar/spelling issues in documentation

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">📋Review Checklist📋</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the project's [branching documentation](../../Documentation/Branching.md).

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] Pull request title matches the title of the associated issue.

- [x] A `preview` label has been added to the pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] Project assigned matches the project of the associated issue.

- [x] Milestone assigned matches the milestone of the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out, and let's discuss it.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: Carbonate/IMessage.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ public interface IMessage
     /// Deserializes the message to the type <typeparamref name="T"/>.
     /// </summary>
     /// <param name="onError">The action to invoke if an exception occurs.</param>
-    /// <typeparam name="T">The type to deserialize the message into.</typeparam>
```
Comment (CalvinWilkinson) [inline Carbonate/IMessage.cs:16]: Please revert this back based on the conversation we had about prepositions.

File: Carbonate/IReactable.cs [hunk 2]
```diff
@@ -35,15 +35,15 @@ public interface IReactable : IDisposable
...
-    /// <param name="eventId">The ID of the event to push the notification to.</param>
+    /// <param name="eventId">Where to push the ID event notification.</param>
     /// <typeparam name="T">The type of data to push.</typeparam>
     void Push<T>(in T data, Guid eventId);
 
     /// <summary>
     /// Pushes a single notification using the given <paramref name="message"/>.
     /// </summary>
     /// <param name="message">The message that contains the data to push.</param>
-    /// <param name="eventId">The ID of the event to push the notifications to.</param>
```
Comment (CalvinWilkinson) [inline Carbonate/IReactable.cs:46]: Refactor to below:

```suggestion
    /// <param name="eventId">The ID of the event where the notification will be pushed.</param>
```

File: Carbonate/Services/JsonSerializer.cs [hunk 3]
```diff
@@ -16,7 +16,7 @@ internal sealed class JsonSerializer : ISerializer
     /// <inheritdoc/>
     public string Serialize<T>(T? value)
     {
-        // TODO: Set these options to only be indented if in debug mode
+        // TODO: Set these options to only be indented if in debug mode  ******Do you still need this TODO?********
```
Comment (CalvinWilkinson) [inline Carbonate/Services/JsonSerializer.cs:19]: I do not need the TODO anymore.  I am going to push some changes on your branch to quickly fix this.

**Review by CalvinWilkinson — CHANGES_REQUESTED** [review]

**Review by CalvinWilkinson — APPROVED** [review]

**Comment by kselena** [comment]
@CalvinWilkinson are you going to merge this PR?

**Comment by CalvinWilkinson** [comment]
> @CalvinWilkinson are you going to merge this PR?

Yes.  Good catch.  I forgot about this.  Go head and merge it if you can. :)
